bedrock/firefox/views.py

```diff
@@ -528,5 +528,8 @@ class HelloStartView(LatestFxView):
     def get_context_data(self, **kwargs):
         ctx = super(HelloStartView, self).get_context_data(**kwargs)
         incoming = self.request.GET.get('incomingConversation') or 'none'
+        source = self.request.GET.get('utm_content') or 'none'
+        source = source.replace('hello-tour_OpenPanel_', '')
```

Comment:
This `source.replace` doesn't seem to be working for me, I still see `utm_content=hello-tour_OpenPanel_firstrun` in the URL for example. Also, I'm not sure if this is the best method to do this, or if it's strictly necessary? 

Maybe @pmclanahan can advise?